### PR TITLE
[MRG] troubleshooting mne c install libxp6

### PR DESCRIPTION
**Pull request by agramfort**

### CODE REVIEW

**Comment by agramfort** [comment]
here is how it looks like:

https://9854-1301584-gh.circle-artifacts.com/0/html/install_mne_c.html#troobleshooting

GTG from my end.

File: doc/install_mne_c.rst [hunk 1]
```diff
@@ -167,6 +167,29 @@ listed in :ref:`CIHDGFAA`.
     |                         | values, see :ref:`mne_process_raw`.        |
     +-------------------------+--------------------------------------------+
 
+
+Troobleshooting
```
Comment (larsoner) [inline doc/install_mne_c.rst]: Troubleshooting

File: doc/install_mne_c.rst [hunk 2]
```diff
@@ -167,6 +167,29 @@ listed in :ref:`CIHDGFAA`.
...
+.. code-block:: console
+
+    $ mne_process_raw
+    mne_process_raw: error while loading shared libraries: libXp.so.6: cannot open shared object file: No such file or directory
+
+If you are using Ubuntu Linux and you have admin access to the machine you can do:
+
+.. code-block:: console
+
+    $ sudo add-apt-repository "deb http://security.ubuntu.com/ubuntu precise-security main"
```
Comment (larsoner) [inline doc/install_mne_c.rst]: it's probably eaiser just to do a `wget` on the `.deb` and a `sudo apt install ...`, but if this works it's probably fine, too.

Also I think there is often a missing `libgfortran.so.3` (?) that could be listed here, does it get pulled in as a dependency by this? If not we can deal with it next time someone brings it up.

**Comment by agramfort** [comment]
>
> it's probably eaiser just to do a wget on the .deb and a sudo apt install
> ..., but if this works it's probably fine, too.
>
> Also I think there is often a missing libgfortran.so.3 (?) that could be
> listed here, does it get pulled in as a dependency by this? If not we can
> deal with it next time someone brings it up.
>

indeed but I think it's libgfortran.so.1 which is usually missing. If you
have a clean fix for this one I am interested....  and yes it should be in
this section.

**Comment by massich** [comment]
LGTM. I tried to reproduce the missing `libgfortran.so.1` to do everything in a single PR. But it does not break in the machine I tried. (despite I've seen the error you are talking about before).

I'm +1 in merging this one as it is, and then do another PR with the other work around.

**Comment by agramfort** [comment]
ok thx. @larsoner merge if you're happy

**Comment by agramfort** [comment]
ok thanks. @larsoner merge if you're happy

**Comment by agramfort** [comment]
ok thanks. @larsoner merge if you're happy

**Comment by larsoner** [comment]
Thanks @agramfort

**Pull request merged**